examples/render-opengl/src/opengl.rs

```diff
@@ -40,12 +41,18 @@ pub fn launch_opengl_window() -> Result<App, Box<dyn Error>> {
 		.with_title("Render Inochi2D Puppet (OpenGL)");
 
 	let (window, gl_config) = glutin_winit::DisplayBuilder::new()
-		.with_preference(ApiPrefence::FallbackEgl)
+		.with_preference(ApiPreference::FallbackEgl)
 		.with_window_builder(Some(window_builder))
 		.build(&events, <_>::default(), |configs| {
 			configs
-				.filter(|c| c.srgb_capable())
-				.max_by_key(|c| c.num_samples())
+				.filter(|c| match c {
+					Config::Egl(c) => c.srgb_capable(),
+					Config::Glx(c) => c.srgb_capable(),
+				})
+				.max_by_key(|c| match c {
+					Config::Egl(c) => c.num_samples(),
+					Config::Glx(c) => c.num_samples(),
+				})
```

Comment:
So why this change? I just tried and `c.srgb_capable()` is still available on the root data structure.

Reply:
not in my version. I feel like we're testing on different libraries or compilers. I'm fixing up the other platforms rn

Reply:
I think I had the wrong version of the import

Reply:
Ah, turns out it was a wrong import, alright :v